.github/workflows/release_linux_x86_64.yml

```diff
@@ -98,3 +99,50 @@ jobs:
         export ORT_MAX_ML_OPSET_SUPPORTED_VERSION=3
         export ORT_MAX_ONNX_OPSET_SUPPORTED_VERSION=19
         pytest
+
+  
+  
+  github-release:
```

Comment:
Would it make sense to upload signed packages in the `Upload wheel to PyPI weekly` step?